apps/README.md

```diff
@@ -189,6 +189,52 @@ fetching files served by the test runner, prefix the file path with
 document.write('<audio src="/base/test/audio/assets/win.mp3"/>');
 ```
 
+#### UI Component Style Guide ####
+
+We use `react-storybook` to generate a ui component style guide that you can use
+to discover what components are available to reuse as you build new
+features. You can also use the style guide to more easily develop new components
+without having to run all of code.org.
+
+To view the styleguide run
+
+```
+npm run storybook
```

Comment:
does it stay running and you hit ctrl-c to terminate, or does it just execute something and then the regular dashboard web server serves up this new page?

if the latter, should this also be part of `rake build`?

also i think we had an `npm` invoked command from the root of our source tree that listed every single npm command that we support, but i can't remember what it was.  @islemaster might know.  If so, might be worth listing this command there, too.